packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+# Release Process for Base, Celocli, ContractKit, DAppKit, and Utils
+
+## Versioning
+
+Use the standard MAJOR.MINOR.PATCH semantic versioning scheme described at [semver.org](http://semver.org).
+
+New releases can be expected as follows:
+  * Major releases: approximately yearly 
+  * Minor releases: approximately 8 times a year
```

Comment:
Are you thinking we will publish a minor release after every milestone?

Reply:
yah that's what i was imagining. Wdythink? Ccing @barbaraliau and @eelanagaraj too for thoughts!

Reply:
I think once a milestone makes sense for minor releases, just to have a consistent opportunity to push improvements. (Maybe adding `(once per milestone)` or `(once every 6 weeks)` next to 8x per year helps provide clarity?)

Is there a particular reason we want yearly major releases? Is it just to have a once-yearly opportunity to bundle together + push all backwards incompatible changes at once?

Reply:
I think a minor release after every milestone makes sense. That sets a good cadence and fits into the current workflow. With regards to major releases once a year, it's an approximation and it does typically tend to take a long time to build out a major release + documentation + community feedback and communication. The timing can fluctuate, but it's a good starting point.

Reply:
^yah exactly, more just an estimate/starting point